src/nnet3/nnet-am-decodable-simple.h

```diff
@@ -392,8 +395,247 @@ class DecodableAmNnetSimpleParallel: public DecodableInterface {
 };
 
 
+struct IntPairHasher {
```

Comment:
This stuff shouldn't be in nnet-am-decodable-simple.{h,cc}, please move to something like nnet-batch-compute.{h,cc}